src/scenes/home/home.js

```diff
@@ -165,6 +168,15 @@ class Home extends Component {
        return false;
      }
    }
+   
+   checkIfIE = () => {
+    let ua = window.navigator.userAgent;
+    let msie = ua.indexOf('MSIE');
+    const trident = ua.indexOf('Trident/');
+    if (msie > 0 || trident > 0) {
+      this.setState({ showUpgradeBrowserModal: true });
+    }
+  }
```

Comment:
Consider the following way of writing this function:

```
import 'getValue' from 'lodash/get';

checkIfIE = () => {
  const userAgent = getValue(window, 'navigator.userAgent', '');

  const hasIEUserAgent = userAgent.includes('MSIE' || 'Trident');

  if (hasIEUserAgent) {
    this.setState({ showUpgradeBrowserModal: true });
  }
}
```

Reply:
agree with above comment - `ua` and `msie` have no reason to be defined w/ `let` since they are not being reassigned anywhere within scope.

Reply:
@kylemh and @jjhampton Thanks for the comments. I am learning a lot. But one question, any specific reasons for using lodash?

Reply:
@erathinam That's a good question, and @kylemh can probably answer this better than I.

Seems like the lodash `get` method can handle errors here and also assign a default `userAgent` value of `''` (empty string) if the property `window.navigator` resolves to undefined.

Another way I could see writing this (without using an external dependency) could be with the JavaScript logical OR operator, such as `const userAgent = window.navigator.userAgent || ''`, but then there's also the risk of reference errors in case `window` or `window.navigator` are somehow null/undefined, and you might end up having to write something like `const userAgent = window && window.navigator && window.navigator.userAgent || ''` which can get messy.

Reply:
Got it. Thanks for the info @jjhampton .